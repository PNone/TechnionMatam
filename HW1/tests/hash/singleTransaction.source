daniel baraa 100 20:20
