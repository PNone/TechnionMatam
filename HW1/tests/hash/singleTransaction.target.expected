07ed4b218fcfe1332544